201-front-door-standard-premium-app-service-public/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "appName": {
+            "value": "GEN-UNIQUE"
+        },
+        "appServicePlanSkuName": {
+            "value": "S1"
```

Comment:
for any hardcoded param values can we move them to defaultValues in the template? Also (see comments in template file) can remove some of these when you add other defaultValues - makes it easier to deploy from the repo.

Reply:
Done. The parameters file is now empty - I assume that's what you wanted?